source/extensions/filters/network/redis_proxy/command_splitter_impl.h

```diff
@@ -45,18 +46,18 @@ class Utility {
  * All command level stats. @see stats_macros.h
  */
 // clang-format off
-#define ALL_COMMAND_STATS(COUNTER, HISTOGRAM)                                                      \
+#define ALL_COMMAND_STATS(COUNTER)                                                                 \
   COUNTER(total)                                                                                   \
   COUNTER(success)                                                                                 \
-  COUNTER(error)                                                                                   \
-  HISTOGRAM(latency)
+  COUNTER(error)
 // clang-format on
 
 /**
  * Struct definition for all command stats. @see stats_macros.h
  */
 struct CommandStats {
-  ALL_COMMAND_STATS(GENERATE_COUNTER_STRUCT, GENERATE_HISTOGRAM_STRUCT)
+  ALL_COMMAND_STATS(GENERATE_COUNTER_STRUCT)
+  Envoy::Stats::Histogram& latency_;
```

Comment:
can you just make sure we have test-cases covering millis and micros for this?

Reply:
There are, it's in `test/extensions/filters/network/redis_proxy/command_splitter_impl_test.cc`,  all test check the millis and `RedisSingleServerRequestWithLatencyMicrosTest` specifically check the `micros`.